backend/core/views_auth.py

```diff
@@ -51,7 +51,7 @@ def callback(request):
                 username = username,
                 password = username,
                 email = userinfo['email'],
-                is_active = False, 
+                is_active = True if PRODUCTION else False, 
```

Comment:
Eh, you import `DEVELOPMENT`, but reference `PRODUCTION`. Does that work?

```suggestion
                # On QA/Staging users are not automatically granted access
                is_active = False if DEVELOPMENT else True, 
```

Reply:
Meant to do the opposite - in all environments except PROD users needs to be whitelisted. Will update the import, would have failed as soon as OAuth whitelist is implemented